### PR TITLE
Bug 1218520 - Radio favourite/bookmark test

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8c606b86e6ec78cc684b8ecc29e623e83d10c461

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=327211d015af3944d30af4c2736ec1431d6b18b8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=50ee80905cb56d299712d3ad1809f79f8606ed29

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0e50d6ee4c7022747915f1ac9eca9e934146b5e5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8d2cc02705cfce2233218eaedaf64c3036756781

File: apps/fm/test/marionette/fm_test.js [hunk 1]
```diff
@@ -72,6 +72,46 @@ marionette('FM Radio test', function() {
   });
 
   suite('FM Radio favourites', function() {
+    test('Test favouriting', function() {
```
Comment (justindarc) [inline apps/fm/test/marionette/fm_test.js]: Unnecessary "u" in "favoriting".. Hahaha.. j/k!

**Pull request merged**